lib/ex_doc/formatter/html.ex

```diff
@@ -15,10 +15,21 @@ defmodule ExDoc.Formatter.HTML do
   def run(module_nodes, config) when is_map(config) do
     config = normalize_config(config)
     output = Path.expand(config.output)
-    File.rm_rf! output
-    :ok = File.mkdir_p output
+    build = Path.join(output, ".build")
+
+    if File.exists? build do
+      File.stream!(build)
+      |> Enum.map(fn(line) ->
```

Comment:
My recommendation is to avoid using pipe if you are doing it only once, so I would rather write this one as `Enum.each(File.stream!(build), fn...)`. I would also suggest moving the whole if check to its own function. :)